Backend/UserService.cs

```diff
@@ -119,5 +119,32 @@ public async Task<IdentityResult> UpdateUserAsync(User userToUpdate)
         {
             return await _userManager.UpdateAsync(userToUpdate);
         }
+
+        public UserInformations GetUserFromLDAP(string login, string password)
+        {
+            login += "@fri.uniza.sk";
+
+            OptionsLdap options = new OptionsLdap
+            {
+                SecureSocketLayer = false,
+                BaseDN = "DC=fri,DC=uniza,DC=sk",
+                HostName = "ldapbdc.fri.uniza.sk",
```

Comment:
Should be somwhere in the configuration, not directly in the code

Reply:
Ok I will look into this